trogdor/src/main/java/org/apache/kafka/trogdor/rest/ErrorResponse.java

```diff
@@ -22,46 +22,29 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-import java.util.Objects;
-
 /**
  * An error response.
  */
-public class ErrorResponse {
-    private final int code;
-    private final String message;
-
+public record ErrorResponse(int code, String message) {
     @JsonCreator
     public ErrorResponse(@JsonProperty("code") int code,
                          @JsonProperty("message") String message) {
         this.code = code;
         this.message = message;
     }
 
+    @Override
```

Comment:
nicely  done, thanks. Until taken care by plugin it can happen anytime leaving this task kind of open. Offering to apply thins kind of conventions via rewrite. 

- https://docs.openrewrite.org/recipes/staticanalysis/missingoverrideannotation